lms/djangoapps/grades/models.py

```diff
@@ -243,6 +244,7 @@ def save_grade(cls, **kwargs):
         except IntegrityError:
             is_created = False
         if not is_created:
+            grade = cls.objects.get(user_id=user_id, course_id=usage_key.course_key, usage_key=usage_key)
```

Comment:
@jcdyer Does this seem correct to you? Before, when hitting this line, `grade` had not actually been created and could not be referred to in the next line.